client/src/components/ratings-reviews/Reviews.jsx

```diff
@@ -1,14 +1,18 @@
 /* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import axios from 'axios';
+import Ratings from './Ratings.jsx';
 import ReviewList from './ReviewList.jsx';
 
 // const getReviews = (review) => (<ReviewElement review={review}/>);
 
 class Reviews extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {'data': [{'body': 1}]};
+    this.state = {
```

Comment:
I was researching what we talked about before. This solution works for now, but in the future if you have some kind of data coming in and it's not in the structure you want, you will encounter undefined. So in the future, I think to handle this issue, defaultProps or ternary/if statements (if something is defined, don't render) will be a better solution longterm.

Reply:
This is good advice. Will refactor with this in mind if there's time.